src/raster/r.mapcalc.tiled/r.mapcalc.tiled.py

```diff
@@ -171,6 +171,11 @@ def main():
                     "No tile width or height provided, default tile size set: {h} rows x {w} cols."
                 ).format(h=height, w=width)
             )
+    else:
+        if width:
+            width = int(width)
+        if height:
+            height = int(height)
```

Comment:
Thanks for catching this, I suggest to move this before the `if` and then you can additionally remove line 160-161 and 165-166.